pkg/asset/store/store.go

```diff
@@ -192,7 +193,7 @@ func (s *storeImpl) saveStateFile() error {
 // fetch populates the given asset, generating it and its dependencies if
 // necessary, and returns whether or not the asset had to be regenerated and
 // any errors.
-func (s *storeImpl) fetch(a asset.Asset, indent string) error {
+func (s *storeImpl) fetch(ctx context.Context, a asset.Asset, indent string) error {
```

Comment:
Should we make a child context here? This context would be directly cancellable by the caller of `Fetch` is that the intention ?